src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -31,6 +31,12 @@
             <argument type="service" id="translator.selector" />
         </service>
 
+        <service id="translator.formatter" alias="translator.formatter.default" />
```

Comment:
This is not needed (it's set anyway in the extension due to our default values).